hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -55,11 +55,6 @@ objects:
   spec:
     additionalCertificateAuthoritiesSecretRef:
     - name: ${ADDITIONAL_CERTIFICATE_AUTHORITY}
-    managedDomains:
-    - aws:
-        credentialsSecretRef:
-          name: ${EXTERNAL_DNS_AWS_CREDS}
-      domains:
-      - ${EXTERNAL_DNS_MANAGED_DOMAIN}
+    managedDomains: "${{MANAGED_DOMAINS}}"
```

Comment:
The new parameter should be also added to the parameters section.
I suggest to add it with a default value. The reason is that this PR will be the first to be merged, while the saas repo still does not contain the parameter.

Reply:
done